developer-docs-site/docs/tutorials/run-a-fullnode.md

```diff
@@ -72,43 +72,66 @@ You can also use Docker to configure and run your FullNode.
 4. Download the devenet [genesis][devnet_genesis] and [waypoint][devnet_waypoint] files into that directory.
 5. Run docker-compose: `docker-compose up`.
 
-## Understand and verify the correctness of your FullNode
+## Verify the correctness of your FullNode
 
-### Initial synchronization
-During the initial synchronization of your FullNode, there may be a lot of data to transfer.
+### Verify initial synchronization
+During the initial synchronization of your FullNode, there may be a lot of data to transfer. Progress can be monitored
+by querying the metrics port to see what version your node is currently synced to. Run the following command:
 
-* Progress can be monitored by querying the metrics port `curl 127.0.0.1:9101/metrics 2> /dev/null | grep aptos_state_sync_version | grep type`, which will print out several counters:
-  * `aptos_state_sync_version{type="committed"}` -- the latest (blockchain) version that is backed by a signed commitment (ledger info) from the validators
-  * `aptos_state_sync_version{type="highest"}` -- the highest or latest known version, typically the same as target
-  * `aptos_state_sync_version{type="synced"}` -- the latest blockchain version available in storage, it might not be backed by a ledger info
-  * `aptos_state_sync_version{type="target"}` -- the state sync's current target ledger info version
-* The Executor component will update the output log by showing that 1000 blocks are committed at a time:
+```
+curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type=\"synced\"}"
+```
 
-  ```
-  fullnode_1  | INFO 2020-09-28T23:16:04.425083Z execution/executor/src/lib.rs:534 sync_request_received {"local_synced_version":633750,"name":"chunk_executor","first_version_in_request":633751,"num_txns_in_request":250}
-  fullnode_1  | INFO 2020-09-28T23:16:04.508902Z execution/executor/src/lib.rs:580 sync_finished {"committed_with_ledger_info":false,"name":"chunk_executor","synced_to_version":634000}
-  ```
+This command will output the current synced version of your node. For example:
+```
+$ curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type=\"synced\"}"
+aptos_state_sync_version{type="synced"} 71000
+```
+  
+Compare the synced version returned by this command (e.g., `71000`) with the `Current Version` (latest) shown on the
```

Comment:
nit: I am wondering if there is a counter we can obtain for this and print it here instead of manually retrieving this from status page?

Reply:
Probably -- we can do whatever the status page is doing and migrate it :) This is just a quick fix for now.

Reply:
Makes sense. Maybe we can follow up in a separate issue. Might be a good first onboarding task :)